cpp/CMakeLists.txt

```diff
@@ -104,6 +104,13 @@ set(RMM_LOGGING_LEVEL "INFO" CACHE STRING "Choose the logging level.")
 set_property(CACHE RMM_LOGGING_LEVEL PROPERTY STRINGS "TRACE" "DEBUG" "INFO" "WARN" "ERROR" "CRITICAL" "OFF")
 message(VERBOSE "CUML_CPP: RMM_LOGGING_LEVEL = '${RMM_LOGGING_LEVEL}'.")
 
+if(BUILD_CUML_TESTS OR BUILD_PRIMS_TESTS)
+  # Needed because GoogleBenchmark changes the state of FindThreads.cmake, causing subsequent runs to
+  # have different values for the `Threads::Threads` target. Setting this flag ensures
+  # `Threads::Threads` is the same value in first run and subsequent runs.
+  set(THREADS_PREFER_PTHREAD_FLAG ON)
+endif()
```

Comment:
I think we don't need this anymore now that we've updated everything to GTest v1.10.

Reply:
Could you point me to the changes that you're referring to? If this is fixed then we'll want to remove this special-casing everywhere since we also have this in other repos now.

Reply:
Nevermind, @robertmaynard confirmed we still need this